apps/studio/src/pages/sites/[siteId]/settings/agency.tsx

```diff
@@ -37,6 +36,12 @@ const AgencySettingsPage: NextPageWithLayout = () => {
   })
   const trpcUtils = trpc.useUtils()
   const toast = useToast()
+  const agencySettingsSchema = Type.Pick(
+    SiteConfigSchema,
+    Type.Union([Type.Literal("siteName"), Type.Literal("agencyName")]),
+  )
+  type AgencySettings = Static<typeof agencySettingsSchema>
+  const validateFn = ajv.compile<AgencySettings>(agencySettingsSchema)
```

Comment:
thought: I'm not sure how the persisting to DB part would be done later, but seems like we might need to use `Type.Mapped` rather than `Type.Pick`, and map it such that the `format` is set to `hidden` for the fields that we don't want to show here.

That way, the JSON saved to the backend is still the full JSON value, but the UI is just the 2 fields that you want to show